profile/README.md

```diff
@@ -1,4 +1,17 @@
-## Hi there 👋
+Welcome to Stereum's Github Page 👋
+The Stereum team develops tools to manage the process of setting up & maintaining an Ethereum node simpler without compromising self sovereignty, privacy and flexibility.
+
+An Ethereum node enable its user to use dApps in the most decentralized fashion, to participate in the network by staking $ETH & to obtain/collect their own blockchain data. Stereum aims to cover your use-case as well!
+
+🍄 By using Stereum it is simple to join Ethereum! ⯁
+If you are interested in following our development progress, check out our socials & donate to our GitCoin:
+
+🌐 Website: stereum.net
```

Comment:
Please add links to all of the contacts: https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#links